.github/workflows/widget_wave.yml

```
@@ -0,0 +1,49 @@
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
```

actionlint (1.7.4)

"steps" section is missing in job "build"

(syntax-check)